### ISSUE TITLE
Implement SMS / Text message 2FA provider

### ISSUE THREAD

**Issue by bravegag** [mohiva/play-silhouette#566]
We currently have the TOTP implementation nevertheless it would be nice to have a SMS / Text Message 2FA provider implementation that works as example implementation with some well known provider free development version.

**Comment by akkie** [comment]
Yeah :+1:, would be a nice to have. But this functionality is not on my priority list. A PR is always welcome.
